packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.test.ts

```diff
@@ -265,7 +265,6 @@ describe('BitcoinChainAdapter', () => {
         value: '400',
         wallet,
         chainSpecific: {
-          opReturnData: 'nm, u',
```

Comment:
Tests concerning opReturnData will come in a subsequent PR when the rest of opReturnData is implemented